### PR TITLE
Bypass clickhouse database/sql abstraction layer

### PR DESCRIPTION
**Pull request by serprex**
Remove transactional schema updates because clickhouse isn't transactional

### CODE REVIEW

File: flow/connectors/clickhouse/qrep.go [hunk 1]
```diff
@@ -68,31 +68,23 @@ func (c *ClickhouseConnector) createMetadataInsertStatement(
...
 	}
 	defer rows.Close()
 
-	columnTypes, err := rows.ColumnTypes()
-	if err != nil {
-		return nil, fmt.Errorf("failed to get column types: %w", err)
-	}
-
-	return columnTypes, nil
+	return rows.ColumnTypes(), nil
```
Comment (heavycrystal) [inline flow/connectors/clickhouse/qrep.go:79]: is the output identical across the two functions

Comment (serprex) [reply flow/connectors/clickhouse/qrep.go:79]: Different types, but same for our purposes

File: flow/connectors/clickhouse/qrep.go [hunk 2]
```diff
@@ -68,37 +68,23 @@ func (c *ClickhouseConnector) createMetadataInsertStatement(
...
 func (c *ClickhouseConnector) IsQRepPartitionSynced(ctx context.Context,
 	req *protos.IsQRepPartitionSyncedInput,
 ) (bool, error) {
-	//nolint:gosec
 	queryString := fmt.Sprintf(`SELECT COUNT(*) FROM %s WHERE partitionID = '%s'`, qRepMetadataTableName, req.PartitionId)
 
-	row := c.database.QueryRowContext(ctx, queryString)
+	row := c.database.QueryRow(ctx, queryString)
 
 	var count int
```
Comment (Amogh-Bharadwaj) [inline flow/connectors/clickhouse/qrep.go:89]: For some reason, this is yielding the error:
```
failed to get fetch status of partition: failed to execute query: clickhouse [ScanRow]: (COUNT()) converting UInt64 to *int is unsupported. try using *uint64
```

Comment (Amogh-Bharadwaj) [reply flow/connectors/clickhouse/qrep.go:89]:
```golang
var count uint64
```

is needed

**Review by Amogh-Bharadwaj — APPROVED** [review]
functionally tested

**Pull request merged**